### DISCUSSION TITLE
Allow storing profiles/settings in a selected drive location

### DISCUSSION THREAD

**Discussion by TurtleCrawler** [OrcaSlicer/OrcaSlicer#5901] [Ideas]
I currently use the bambu cloud sync, but I would rather sync my settings to my google drive, or other cloud storage solutions.

I would love to have the ability to change where my settings and profiles are saved to, so I could pick a cloud drive or local network drive folder that I have control over. This would also allow me to get away from relying on the bambu cloud, and make for a more stable experience between devices in my opinion.

**Comment by foreachthing** [comment]
This might help: https://github.com/SoftFever/OrcaSlicer/issues/1027

**Comment by TurtleCrawler** [comment]
That seems like a work around, I would like to see this as a built in feature/setting.

**Comment by YosypC** [comment]
Any update?
